tools/psoxy-test/cli.js

```diff
@@ -31,6 +31,7 @@ const theme = {
     .option('-t, --token <token>', 'Authorization token for GCP')
     .option('-v, --verbose', 'Verbose output', false)
     .option('-z, --gzip', 'Add gzip compression header', false)
+    .option('--save-to-file', 'Save test results to file', false)
```

Comment:
so I would probably prefer if `-s` is alias to this, rather than `--skip`; saving output to a file is a very common use case for our customers, whereas `--skip` is really just for us (dev purpose)